hudi-hadoop-common/src/main/java/org/apache/hudi/storage/hadoop/HoodieHadoopStorage.java

```diff
@@ -58,27 +58,10 @@
 public class HoodieHadoopStorage extends HoodieStorage {
   private final FileSystem fs;
 
-  public HoodieHadoopStorage(HoodieStorage storage) {
```

Comment:
Is this moved to somewhere else?

Reply:
Yeah, I made it the getRawStorage method below

Reply:
Makes sense.